### ISSUE TITLE
Support for .WithMisfireHandlingInstruction

### ISSUE THREAD

**Issue by claudiuconstantin** [DataMicron/quartznet-mongodb#10]
Quartz triggers provide the [.WithMisfireHandlingInstruction](http://www.quartz-scheduler.net/documentation/quartz-2.x/tutorial/simpletriggers.html) functionality for handling job misfires. Does 'quartznet-mongodb' support this, is this part of the trigger serialized?

**Comment by mxurshid** [comment]
Yes, Misfire instructions should work. But there is no test for it. There will need to add tests.

**Comment by claudiuconstantin** [comment]
There are two things here: 
- serializing/deserializing the "MisfireInstruction" property on the Trigger - this works;
- taking the MisfireInstruction into account, updating the next time a job will run according to the specified setting - I'm currently having some issues here and I'm trying to figure out if there's something to fix in the "quartznet-mongodb" implementation or the issue is from somewhere else...
